### DISCUSSION TITLE
Option for citation-location breaks PDF's bibliography (1.3.x)

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#3923] [Q&A] [answered]
I noticed the References section in the PDF of my book project was empty, sometime after upgrading to 1.3.x (using SVG files) this past week. I finally traced down the problem. It's this setting in the _quarto.yml that's documented at https://quarto.org/docs/reference/formats/html.html#references:

```yaml
citation-location: document
```

It wasn't causing the problem prior to 1.3.x IIRC.

I updated my sample repo at https://github.com/fuhrmanator/quarto-test -- just do `quarto preview --to pdf` to see the problem. Comment-out the above config in the .yml to see the references come back (at least on my Windows system and my CI that runs on Ubuntu).

**Comment by dragonstyle** [comment, marked as answer]
This is a bug which I will fix! TY!

**Reply by dragonstyle** [reply]
(fixed here https://github.com/quarto-dev/quarto-cli/commit/4df1cae41f46f3285f3018c602fdeda44c8dcac0 - new build is on the way)
